### DISCUSSION TITLE
Modern Slavery Statement

### DISCUSSION THREAD

**Discussion by clementbiron** [OpenTermsArchive/terms-types#80] [Qualified]
### What are some documents containing these terms?

| Example | Service Name | URL | Title | Jurisdiction(s) |
|:----|:----|:----|:------|:-------------|
| 1 | SailPoint | [URL 🔗](https://www.sailpoint.com/legal/modern-slavery-statement/) | Modern Slavery Statement | 🌐 |
| 2 | Societe Generale | [URL 🔗](https://www.societegenerale.com/sites/default/files/documents/CSR/modern-slavery-act.pdf) | Statement on modern slavery and human trafficking | 🌐 |
| 3 | Amazon | [URL 🔗](https://sustainability.aboutamazon.com/modern-slavery-statement.pdf) | Modern Slavery Statement | 🌐 |
| 4 | Acas | [URL 🔗](https://www.acas.org.uk/acas-modern-slavery-and-human-trafficking-statement) | Modern slavery and human trafficking statement | 🌐 |
| 5 | British Council | [URL 🔗](https://www.britishcouncil.org/statement-slavery) | Modern Slavery Statement | 🌐 |
| 6 | ICO | [URL 🔗](https://ico.org.uk/about-the-ico/modern-slavery-statement/) | Modern Slavery Statement | 🌐 |
| 7 | Grant Thornton | [URL 🔗](https://www.grantthornton.global/en/modern-slavery-statement/) | Modern Slavery Statement | 🌐 |

### What could be some alternative names for this type?

Modern slavery and human trafficking statement

### Whose rights and duties are defined in these terms?

employees, workers, suppliers, business partners, customers, investors, regulatory authorities

### What is the object of commitment in these terms?

measures taken to prevent modern slavery in activities and supply chains

### Are there any relevant references for this type?

UK [Modern Slavery Act 2015](https://en.wikipedia.org/wiki/Modern_Slavery_Act_2015)
AUS [Modern Slavery Act 2018](https://www.legislation.gov.au/C2018A00153/asmade/text)

**Comment by MattiSG** [comment]
I don't find the alternative name “Modern slavery and human trafficking statement” very relevant if the main one is “Modern slavery statement”. I suggest instead “Human trafficking statement” as an alternative.

**Comment by MattiSG** [comment]
I don't understand in which way such terms define rights and duties for “investors” and “regulatory authorities”.
